src/neo/Wallets/Wallet.cs

```diff
@@ -348,6 +348,8 @@ private Transaction MakeTransaction(DataCache snapshot, byte[] script, Signer[]
                 }
 
                 tx.NetworkFee = CalculateNetworkFee(snapshot, tx);
+                if (tx.NetworkFee - (tx.Size * NativeContract.Policy.GetFeePerByte(snapshot)) > MaxVerificationGas)
```

Comment:
You can't calculate the size here because `Witnesses` is null.

Reply:
Agree, we should do that in neo-node